### ISSUE TITLE
Backspace not working

### ISSUE THREAD

**Issue by Fanatic191** [ExXTreMe315/native-tuner#7]
If i would like to go back with one menu, the "backspace" as backward/one step back not working. Thank you, if you can help us. :)

**Comment by ExXTreMe315** [comment]
Yes, I know the problem with AltV V14 and want to fix it tomorrow. 
The problem with the backspace is new for me and I will take a look at it.

Thanke you for your Issue

**Comment by Fanatic191** [comment]
Thank you so much for your reply and thank you for your hard work with that. 

I tried many times, checked the keys too to edit, if it is missing or something, but as i do not have enough knowledge about that, i did not change it on my own. 

A warm welcome
Fanatic 🥰

**Comment by ExXTreMe315** [comment]
Hey I tried "backspace" as backward/one step and in my try it worked. 
So do you got some errors in your console or another script that blocks the backspace.

LG ExXTreMe
